docs/install_guides/installing-openvino-apt.md

```diff
@@ -1,164 +1,133 @@
-| [Demo Applications](@ref omz_demos) | A set of console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use cases. |
-| Additional Tools                                   | A set of tools to work with your models including [Accuracy Checker utility](@ref omz_tools_accuracy_checker), [Post-Training Optimization Tool Guide](@ref pot_README), [Model Downloader](@ref omz_tools_downloader) and other  |
-| [Documentation for Pre-Trained Models ](@ref omz_models_group_intel)                                   | Documentation for the pre-trained models available in the [Open Model Zoo repo](https://github.com/openvinotoolkit/open_model_zoo).  |
-| Deep Learning Streamer (DL Streamer)   | Streaming analytics framework, based on GStreamer\*, for constructing graphs of media analytics components. For the DL Streamer documentation, see [DL Streamer Samples](@ref gst_samples_README), [API Reference](https://openvinotoolkit.github.io/dlstreamer_gst/), [Elements](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/Elements), [Tutorial](https://github.com/openvinotoolkit/dlstreamer_gst/wiki/DL-Streamer-Tutorial). |
-
-
-## Install Packages
-
-### Set up the OpenVINO™ Toolkit APT Repository
+- Ubuntu 18.04 long-term support (LTS), 64-bit
+- Ubuntu 20.04 long-term support (LTS), 64-bit
+
+## Install OpenVINO Runtime
+
+### Step 1: Set up the OpenVINO™ Toolkit APT repository
+
+
+1. Install the GPG key for the Repository
+
+    a. Download the [GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB](https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB). You can also use the following command:
+      ```sh
+      wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+      ```    
+    b. Add this key to the system keyring:
+      ```sh
+      sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+      ```
+      > **NOTE**: You might need to install GnuPG: `sudo apt-get install gnupg`   
+      
+2.	Add the repository via the following command:
+
+   * On Ubuntu 18
+   ```sh
+   echo "deb https://apt.repos.intel.com/openvino/2022/bionic all main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+   ```
+   
+   * On Ubuntu 20
+   ```sh
+   echo "deb https://apt.repos.intel.com/openvino/2022/focal all main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+   ```  
+
+3.	Update the list of packages via the update command:
+   ```sh
+   sudo apt update
+   ```       
+   
+4.	Verify that the APT repository is properly set up. Run the apt-cache command to see a list of all available OpenVINO packages and components:
+   ```sh
+   apt-cache search openvino
+   ```   
+   
+
+### Step 2: Install OpenVINO Runtime using the APT Package Manager
+
+Intel® OpenVINO™ Toolkit will be installed in: `/opt/intel/openvino_<VERSION>.<UPDATE>.<PATCH>`
+    
+A symlink will be created: `/opt/intel/openvino_<VERSION>`
```

Comment:
Above (line 44) 2022 is explicitly specified, so is it necessary to keep this generic here? Presumably the document will have to be updated for 2023 anyway, so <VERSION> could be replaced with 2022 everywhere?

Reply:
Thanks. Ideally we should use variables throughout a same page, but I can understand for line 39 and line 44, if we use <version> instead, we may need to use an example to describe what exactly that means, which would make this case unnecessarily more complicated. But for <VERSION>.<UPDATE>.<PATCH>, since we may need to update several times during each release, a generic format would be better and we have an example below. To conclude, I would like to keep the current formatting as it is. :)